src/main/java/ca/on/oicr/gsi/dimsum/service/filtering/CaseFilterKey.java

```diff
@@ -66,6 +67,10 @@ public Function<String, Predicate<Requisition>> requisitionPredicate() {
     return string -> requisition -> true;
   }
 
+  public Function<String, Predicate<TestTableView>> testTableViewPredicate() {
+    return string -> testTableViewPredicate -> true;
```

Comment:
You'll have to override this for `PENDING`, similar to `testPredicate`. You can add methods to `PendingState` to reuse the existing code for handling `Test`s. e.g.

```
public Predicate<TestTableView> testTableViewPredicate() {
  return view -> testPredicate.test(view.getTest());
}
```

Reply:
What is the difference between the code above and using `return view -> qualifytest(view.getTest())`?

Reply:
They'd have the same effect, though your suggestion is slightly better since it's more direct

Reply:
@djcooke Also, wouldn't this cause an error when trying to filter by `PendingState` that uses requisition (ex. final report)? Should I add `qualifytest` for those states and return false?